styles/adoptadog-stylesheet.css

```diff
@@ -0,0 +1,153 @@
+header {
+  background-image: url('../images/header-image.jpg');
+  padding: 3em;
+  height: 20em;
+  color: white;
+  font-family: 'Open Sans', sans-serif;
+}
+.header-h1 {
+  color: lime;
+  font-family: 'Courgette', cursive;
+}
+.nav-header {
+  font-family: "Arial Black", Gadget, sans-serif;
+  height: 1rem;
+  background-color: whitesmoke;
+  padding: 1em 10em 2em;
+}
+.nav-link {
+  text-decoration: none;
+}
+.bold {
+  font-family: 'Open Sans', sans-serif;
+}
+h1 {
+  font-family: "Times New Roman", Times, serif;
+  color: rgb(0, 51, 102);
+  font-weight: bold;
+}
+h2 {
+  font-family: "Times New Roman", Times, serif;
+  color: rgb(0, 51, 102);
+  font-weight: bold;
+  font-size: 24px;
+  margin: 0px 0px;
+}
+h3 {
+  color: rgb(0, 51, 102);
+  font-family: "Times New Roman", Times, serif;
+  font-size: 18px;
+  margin: 0.3em 0px;
+}
+p {
+  font-family: Arial, Helvetica, sans-serif;
+}
+p.large {
+  font-size: 16px;
+}
+p.footer {
+  font-family: 'PT Sans', sans-serif;
+}
+.quote {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+.aligncenter {
+  text-align: center;
+}
+.grid-container {
+  display:grid;
+  grid-gap: 3em 3em;
+  grid-template-columns: auto auto auto auto;
+  justify-content: space-evenly; /*align grid items, not text*/
+  padding: 1em 12em;
+  font-size: 12px;
+}
+.grid-item-1 {
+  grid-area: 1/1/1/4;
+}
+.grid-item-2 {
+  text-align: right;
+}
+.grid-item-3 {
+  grid-area: 1/1/1/3;
+}
+.grid-item-4 {
+  grid-area: 1/3/1/4;
+}
+.grid-item {
+  display:grid;
+  flex-direction: column;
+  border: 2px solid whitesmoke;
+}
+.grid-text {
+  padding: 0.5em 1em 1em 1em;
+}
+footer {
+  background-color: rgb(0, 51, 102);
+  text-align: center;
+  padding: 1em 1em 1em 1em;
+  color: white;
+  font-family: 'PT Sans', sans-serif;
+  padding: 5em 20em 5em;
+}
+a.footer {
+  text-decoration: none;
+}
+form {
+  padding: 1em 10em;
+}
+label {
+  color: rgb(0, 51, 102);
+  font-family: "Times New Roman", Times, serif;
+  font-size: 18px;
+  font-weight:bold;
+}
+input[type=text], select {
+  width: 65%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  margin-bottom:20px;
+}
+input[type=email], select {
+  width: 65%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  margin-bottom:20px;
+}
+input[type=number], select {
+  width: 65%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  margin-bottom:20px;
+}
+input[type=submit] {
+  min-width:20%;
+  width: 20%;
+  background-color: #77C425;
+  color: white;
+  padding: 14px 20px;
+  margin: 6px 0;
```

Comment:
Keep in mind the next assignment requires to add basic responsiveness. This means px are no longer acceptable units for margins, font etc. Use % em or rem instead.